pkg/registry/apis/provisioning/repository_controller.go

```diff
@@ -304,6 +305,23 @@ func (rc *RepositoryController) process(item *queueItem) error {
 			}
 		}
 
+		// Make sure only one repository is targeting the root folder
```

Comment:
shouldn't this be done in the admission? shouldn't we block admission?

Reply:
🤔  yes that could work